lab-abigail/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "lab-abigail",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-03-fs-readfile ======",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./lib"
```

Comment:
Good job remembering a ``lint`` script, but in order for someone to do ``npm run lint`` in their terminal, you still need to download ``eslint`` as a dev dependency, i.e. type ``npm i -D eslint`` in your terminal. No points lost for this, however, just an FYI for the future.